test/unit/model/reference/reference.utils.test.ts

```diff
@@ -242,4 +248,30 @@ describe('deserializeReferenceAnnotatedValue', () => {
       },
     }
     `);
-});
\ No newline at end of file
+});
+
+describe('serializeReferenceContributorsList', () => {
+  const xmlSerializer = new XMLSerializer();
+
+  it('should serialize an empty contributors list', () => {
+    const xml = serializeReferenceContributorsList('groupType', []);
+    expect(xmlSerializer.serializeToString(xml)).toBe('<person-group person-group-type="groupType"/>');
+  });
+
+
+  it('should serialize individual author', () => {
+    const xml = serializeReferenceContributorsList('author', [{firstName: 'John', lastName: 'Doe'}]);
+    expect(xmlSerializer.serializeToString(xml)).toBe('<person-group person-group-type="author"><name><given-names>John</given-names><surname>Doe</surname></name></person-group>');
+  });
+
+  it('should serialize group contributor', () => {
+    const xml = serializeReferenceContributorsList('author', [{groupName: 'Teenage mutant ninja turtles'}]);
+    expect(xmlSerializer.serializeToString(xml)).toBe('<person-group person-group-type="author"><name><collab>Teenage mutant ninja turtles</collab></name></person-group>');
```

Comment:
Where is the spec for `<collab>` to sit inside of a `<name>` tag? I couldn't find any example of this in the kitchen sink xml

Reply:
[this](https://github.com/elifesciences/XML-mapping/blob/7cfdc3919715cb35940ced2f9a6866182bad2698/elife-00666.xml#L1574) and [this](https://github.com/elifesciences/XML-mapping/blob/7cfdc3919715cb35940ced2f9a6866182bad2698/elife-00666.xml#L1654) were the only examples of this in a reference citation

Reply:
This is what Fred explained a contributor structure could be. Some articles have it.

Reply:
Thanks both. `collab` as child of `person-group` please. It's either `name` (with child elements), or `collab`. 

Example of a book ref w/ `collab` [here](https://elifesciences.org/articles/65066#bib143). xml can be gotten [here](https://elifesciences.org/articles/65066.xml) (search `<ref id="bib143">` after page has loaded).